python/cuml/svm/svm_base.pyx

```diff
@@ -198,6 +198,10 @@ class SVMBase(Base):
         self.epsilon = epsilon
         self.svmType = None  # Child class should set self.svmType
 
+        # Parameter to indicate if model has been correctly fitted
+        # fit_status == -1 indicates that the model is not yet fitted
+        self.fit_status_ = -1
```

Comment:
Definitely simplified from before! Sorry to keep picking on this small bit, but should it just be `fit_completed_ = False`  and use a standard boolean logic rather than -1/0 flags?